### ISSUE TITLE
MariaDB 10.2.2: date tests are failed on labrador

### ISSUE THREAD

**Issue by kou** [mroonga/mroonga#116]
All platforms: http://buildbot.askmonty.org/buildbot/grid?category=main&branch=ks-10.2-Mroonga-6.08

Failed platforms:
- http://buildbot.askmonty.org/buildbot/builders/labrador/builds/7766

Log: http://buildbot.askmonty.org/buildbot/builders/labrador/builds/7766/steps/test/logs/stdio

``` text
----
CURRENT_TEST: mroonga/storage.index_multiple_column_unique_datetime_order_desc
--- /private/var/lib/buildslave/maria-slave/labrador/build/storage/mroonga/mysql-test/mroonga/storage/r/index_multiple_column_unique_datetime_order_desc.result 2016-08-25 12:46:11.000000000 +0300
+++ /private/var/lib/buildslave/maria-slave/labrador/build/storage/mroonga/mysql-test/mroonga/storage/r/index_multiple_column_unique_datetime_order_desc.reject 2016-08-31 04:40:04.000000000 +0300
@@ -7,13 +7,19 @@
 );
 INSERT INTO ranges VALUES (1, "2012-10-25 16:18:29", "9999-12-31 23:59:59");
 INSERT INTO ranges VALUES (2, "1000-01-01 00:00:00", "2012-10-05 16:18:29");
+Warnings:
+Warning    1265    Data truncated for column 'start' at row 1
 INSERT INTO ranges VALUES (3, "9999-12-31 23:59:59", "1000-01-01 00:00:00");
+Warnings:
+Warning    1265    Data truncated for column 'end' at row 1
 INSERT INTO ranges VALUES (4, "1000-01-01 00:00:00", "9999-12-31 23:59:59");
+Warnings:
+Warning    1265    Data truncated for column 'start' at row 1
 SELECT * FROM ranges FORCE INDEX(range_key)
 ORDER BY start DESC, end DESC;
 id start   end
-3  9999-12-31 23:59:59 1000-01-01 00:00:00
+3  9999-12-31 23:59:59 1970-01-01 00:00:00
 1  2012-10-25 16:18:29 9999-12-31 23:59:59
-4  1000-01-01 00:00:00 9999-12-31 23:59:59
-2  1000-01-01 00:00:00 2012-10-05 16:18:29
+4  1970-01-01 00:00:00 9999-12-31 23:59:59
+2  1970-01-01 00:00:00 2012-10-05 16:18:29
 DROP TABLE ranges;
----


Failing test(s):
  mroonga/storage.column_datetime_64bit_before_unix_epoch
  mroonga/storage.column_datetime_64bit_version_56_or_later_out_of_range
  mroonga/storage.index_btree_equal_datetime
  mroonga/storage.index_multiple_column_unique_date_64bit_equal
  mroonga/storage.index_multiple_column_unique_date_index_read
  mroonga/storage.index_multiple_column_unique_date_order_64bit_asc
  mroonga/storage.index_multiple_column_unique_date_order_64bit_desc
  mroonga/storage.index_multiple_column_unique_datetime_index_read
  mroonga/storage.index_multiple_column_unique_datetime_order_asc
  mroonga/storage.index_multiple_column_unique_datetime_order_desc
```

**Comment by kou** [comment]
It seems that 64bit date isn't supported on labrador.
But what OS is used on labrador?

**Comment by kou** [comment]
http://buildbot.askmonty.org/buildbot/buildslaves/labrador

> Mac Mini Server 8GB RAM 2x500GB HD, Lion 10.7, Intel i7-2635QM CPU @ 2.00GHz, x86_64

**Comment by kou** [comment]
It'll be fixed by 2a86a6852453e34f62f27a35735602ac56afc5a5. (Reference is wrong...)
